proto/ibc/core/connection/v1/connection.proto

```diff
@@ -104,3 +104,9 @@ message Version {
   // list of features compatible with the specified identifier
   repeated string features = 2;
 }
+
+// Params defines the set of Connection parameters.
+message Params {
+  // expected time per block, used to enforce block delay.
+  uint64 expected_time_per_block = 1 [(gogoproto.moretags) = "yaml:\"expected_time_per_block\""];
```

Comment:
```suggestion
  uint64 max_expected_time_per_block = 1 [(gogoproto.moretags) = "yaml:\"max_expected_time_per_block\""];
```
thoughts? I like having max since it implies the person choosing the parameter should maybe read more documentation about this and use a value larger than the average block time. 

If I was setting this for my chain and saw `expected_time_per_block`, I'd go to mintscan, see that the hub produces blocks every 7 seconds and then set this to `7s`

I think we could add more comments to this proto definition explaining that the value should be a larger than the average block time. Maybe 3-5 times larger